src/EditorFeatures/Core/Implementation/Classification/SyntacticClassificationTaggerProvider.TagComputer.cs

```diff
@@ -35,12 +35,6 @@ internal partial class SyntacticClassificationTaggerProvider
         /// </summary>
         internal partial class TagComputer
         {
-            // This is how long we will wait after completing a parse before we tell the editor to
-            // re-classify the file. We do this, since the edit may have non-local changes, or the user
-            // may have made a change that introduced text that we didn't classify because we hadn't
-            // parsed it yet, and we want to get back to a known state.
-            private const int ReportChangeDelayInMilliseconds = TaggerConstants.ShortDelay;
```

Comment:
from my convo with Jason, this field becamse useless.  we waited a bit to give our bg system time to parse the doc.  but we're always parsing the doc prior to reporting the change (in order to determine what changed).  so any amount of extra delay is superfluous.